Asterix/wfsc/corrector.py

```diff
@@ -7,12 +7,13 @@
 import numpy as np
 from astropy.io import fits
 
-import Asterix.save_and_read as useful
-import Asterix.optical_systems as OptSy
-from Asterix.estimator import Estimator
-import Asterix.WSC_functions as wsc
-from Asterix.THD_quick_invert import THD_quick_invert
+from .estimator import Estimator
 
+from .thd_quick_invert import THD_quick_invert
+from .wf_control_functions import *
```

Comment:
Please don't do blank imports. It's bad practice and can get confusing.

Reply:
ok, I correct both